sync/rpc/server.go

```diff
@@ -1214,3 +1214,66 @@ func (b *BackupList) BackupDelete(backupID string) error {
 	}
 	return fmt.Errorf("backup not found %v", backupID)
 }
+
+func (s *SyncAgentServer) FileSync(ctx context.Context, req *FileSyncRequest) (e *empty.Empty, err error) {
+	if err != s.syncFilesFromHealtyReplica(req.FromServer, req.Disks) {
+		return nil, err
+	}
+
+	return &empty.Empty{}, nil
+}
+
+func (s *SyncAgentServer) syncFilesFromHealtyReplica(fromServer string, disks []string) (err error) {
+	for _, disk := range disks {
+		if strings.Contains(disk, VolumeHeadName) {
```

Comment:
It's better if we can validate this earlier so we won't encounter it late in the loop.